src/Hal/Report/Html/template/index.php

```diff
@@ -141,7 +141,7 @@
                                 return strcmp($a->name, $b->name);
                             });
                             foreach ($packages as $package) { ?>
-                                <tr<?php if (null !== $package->installed && $package->installed !== $package->latest) { echo ' style="color:orangered"'; }?>>
+                                <tr<?php if (null !== $package->installed && version_compare($package->installed, $package->latest) === -1) { echo ' style="color:orangered"'; }?>>
```

Comment:
If you use a package in beta, alpha, RC (i.e non stable) version of a package and there is no stable version above the one you installed, PhpMetrics should not warn you.